src/views/Profile/CreateForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useProfile } from '../../context/ProfileContext';
+import { useHistory, Redirect } from 'react-router-dom';
+import { createProfile, updateProfile } from '../../services/profiles';
+import ProfileForm from '../../components/ProfileForm/ProfileForm';
+import { useUser } from '../../context/UserContext';
+
+export default function CreateForm({ makingProfile = false }) {
+  const { setProfile } = useProfile();
+  const { user } = useUser();
+  const history = useHistory();
+
+  const handleProfileForm = async ({ name, birthday, bio }) => {
+    try {
+      if (makingProfile) {
+        await createProfile({ name, email: user.email, birthday, bio });
+        <Redirect to="/profile" />;
```

Comment:
This `<Redirect />`would need to be returned. Using the history hook might make more sense here